event-adapter/cmd/provider/provider.go

```diff
@@ -17,44 +17,146 @@ limitations under the License.
 package provider
 
 import (
+	gce "cloud.google.com/go/compute/metadata"
 	"fmt"
 	"github.com/GoogleCloudPlatform/k8s-stackdriver/event-adapter/pkg/provider"
-	stackdriver "google.golang.org/api/monitoring/v3"
-	"k8s.io/client-go/pkg/api"
+	sd "google.golang.org/api/logging/v2"
+	api "k8s.io/client-go/pkg/api/v1"
+	"k8s.io/client-go/rest"
+	"time"
 )
 
+//TODO(erocchi) add implementation to write to Stackdriver
```

Comment:
Maybe "Add implementation to write events to Stackdriver" would be just a little bit clearer?

Reply:
Done